assets/js/handlebars/handlebars.helpers.js

```diff
@@ -486,3 +486,11 @@ goToDetail = function(postId) {
         window.location ="post-detail.html#"+id;
     }
 }
+
+Handlebars.registerHelper('postCheckbox', function() {
+    if (session.user.logged_in) {
+        return new Handlebars.SafeString(
+            '<div class="listing-item-select"><input type="checkbox"></div>'
+        );
+    }
+});
```

Comment:
make sure `postLocked` is added back so the locked state is represented.

Reply:
Done.